### DISCUSSION TITLE
ERROR 400: Redirect_URI_MISMATCH

### DISCUSSION THREAD

**Discussion by Gisselle-Guzman** [cisagov/ScubaGoggles#373] [General]
Hi there, 

Apologies if this is a noob question. But this landed on my desk. Have to make this tool work. I am getting an error 400: redirect_URI_Mismatch error , when I try to run the Scuba Google application. I'll attach pics, I followed the directions from the documentation , so stuck.  Any help will really help :) 

![2024-09-02_18-35_400_URI_ERRO](https://github.com/user-attachments/assets/b7b6bb0c-bc13-4ebd-a943-da249fb9c670)

![2024-09-02_18-41_Authorized_Redirects_URI](https://github.com/user-attachments/assets/c34942cd-b1bb-4526-a4fe-b7718b2fba1b)

**Comment by buidav** [comment]
Add the forward slash at the end of the `http://localhost:8080` + `/`  = `http://localhost:8080/`
The last `/` is important.

![8080](https://github.com/user-attachments/assets/07d8a81d-affe-49a8-a8f5-13eb40ce5138)

**Reply by Gisselle-Guzman** [reply]
Thank you!! Yep that was the issue I was having.
